v1.1-beta/matrixx-digital-commerce/matrixx-sba-events/cnf-testsuite.yml

```diff
@@ -0,0 +1,26 @@
+---
+helm_directory: "sba-5gc-events"
+release_name: "cnf"
+helm_install_namespace: "matrixx"
+helm_values: "--values /home/dwilmes/.mtx/konstruxx/working/values-939239483.yaml\
```

Comment:
@deerowbear @daniel-wilmes generic please for end-users to be able to repeat testing if desired